pkg/scaffold/v2/makefile.go

```diff
@@ -70,11 +70,11 @@ run: generate fmt vet
 	go run ./main.go
 
 # Install CRDs into a cluster
-install: manifests
+install: generate
```

Comment:
Generating go code here is not useful, since we don't compile go code here.

Reply:
Fixed